docs/development-environment-cli-support.md

```diff
@@ -0,0 +1,92 @@
+# Support page for the Development Environment Command line checker
+
+### Usage
+
+```sh
+tools/check-development-environment.sh
+```
+
+## Topics
+
+### Command is available: git
+
+You need GIT for contributing to Jetpack.
+
+### Command is available: n
+
+**n** is a node version manager. Similar and alternative to **nvm**.
+
+### Command is available: php
+
+PHP is needed all around the build process for Jetpack bundles.
+
+### Command is available: phpunit
+
+PHPUnit is the tool that helps us run unit tests for Jetpack.
+
+### Docker images are available
+
+The containers for the Jetpack docker images are created when you ran `yarn docker:up`.
+
+### Docker containers are available
+
+The containers for the Jetpack docker containers are created when you ran `yarn docker:up`.
+
+### Docker containers are running
+
+If you are using the Jetpack Docker Image, make sure you run `yarn docker:up`.
+
+### Docker Is Running
+
+make sure the Docker Daemon is running.
+
+### Node Modules are available
+
+The directory `node_modules` is where Jetpack's JavaScript dependencies live. It gets initialized by doing:
+
+```sh
+yarn
+```
+
+### Node version is proper
+
+For developing Jetpack you need Node > 10 nowadays.
```

Comment:
Maybe we shouldn't mention the version number here, since it's most likely to change over time.